.github/workflows/build.yml

```diff
@@ -66,3 +67,10 @@ jobs:
           else
             echo "There were no changes to commit"
           fi
+          
+      - name: Upload dist folder as artifact on commit failure
```

Comment:
This helps with those using forks to manually upload the correct `dist` changes produced.